android/src/main/java/io/sentry/react/RNSentryTimeToDisplay.java

```diff
@@ -0,0 +1,23 @@
+package io.sentry.react;
+
+import com.facebook.react.bridge.Promise;
+
+import android.view.Choreographer;
+
+import org.jetbrains.annotations.NotNull;
+import io.sentry.SentryDate;
+import io.sentry.SentryDateProvider;
+import io.sentry.android.core.SentryAndroidDateProvider;
+
+public class RNSentryTimeToDisplay {
+    public static void GetTimeToDisplay(Promise promise, SentryDateProvider dateProvider) {
+        Choreographer choreographer = Choreographer.getInstance();
```

Comment:
Shouldn't it be in a try catch block?
accessing Choreographer may result in an exception (we had some reports in the SDK console)